lib/Cake/Test/Case/Model/CakeSchemaTest.php

```diff
@@ -364,6 +364,39 @@ class SchemaCrossDatabaseFixture extends CakeTestFixture {
 }
 
 /**
+ * NonConventionalPrimaryKeyFixture class
```

Comment:
You _could_ add some short explanation what these fixtures are for. So one does not have to look up the related commit.